CO-3345 add note about cooldown and permissions (#9079)
braze-inc/braze-docs: _docs/_user_guide/data/custom_data/custom_events/nested_objects.md

![][6]{: style="max-width:80%;"}

> If new properties are sent in the future they will not be in the schema until it is regenerated. Schemas can be regenerated every 24 hours.

Reviewer: ```suggestion
If new properties are sent in the future, they won't be in the schema until it is regenerated. Schemas can be regenerated every 24 hours.
```
